File: src/scanners/YuzuFlatpakSource.js
const YuzuSource = require("./YuzuSource.js");
const YuzuFlatpakGame = require("../games/YuzuFlatpakGame.js");

const USER_DIR = process.env["HOME"];

class YuzuFlatpakSource extends YuzuSource{

	static name = "Yuzu (Flatpak)";
	static gameClass = YuzuFlatpakGame;

	configPath = `${USER_DIR}/.var/app/org.yuzu_emu.yuzu/config/yuzu/qt-config.ini`;

}

module.exports = YuzuFlatpakSource;

File: gali/sources/yuzu/yuzu_flatpak_source.py
from gali.sources.yuzu.yuzu_flatpak_game import YuzuFlatpakGame
from gali.sources.yuzu.yuzu_source import YuzuSource
from gali.utils.locations import HOME


class YuzuFlatpakSource(YuzuSource):

    name: str = "Yuzu (Flatpak)"
    game_class: type[YuzuFlatpakGame] = YuzuFlatpakGame
    config_path: str = f"{HOME}/.var/app/org.yuzu_emu.yuzu/config/yuzu/qt-config.ini"